specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/virtualMachineImage.json

```diff
@@ -938,6 +988,9 @@
         "automaticOSUpgradeProperties": {
           "$ref": "#/definitions/AutomaticOSUpgradeProperties"
         },
+        "imageDeprecationStatus": {
```

Comment:
this new property should go at the end of the properties list for VirtualMachineImageProperties

Reply:
so it should come after "architecture"

Reply:
Addressed. Will regenerate SDK changes as well.